models/topics-models.js

```diff
@@ -0,0 +1,7 @@
+const db = require('../db/connection')
+
+exports.fetchTopics = () => {
+    return db.query(`
+    SELECT * 
+    FROM topics;`)
+}
```

Comment:
Add a .then here, deal with the rows in the model and send back to the controller